### DISCUSSION TITLE
添加数据

### DISCUSSION THREAD

**Discussion by Extreme-Icer** [Ivysauro/CNRT#28] [Q&A]
吉林省 支持支付宝和微信，并且APP就叫吉事办

**Comment by Ivysauro** [comment]
感谢反馈😀

**Comment by chenw23** [comment]
已复核并补全于 PR #153。吉林省政务服务和数字化建设管理局 2025-04-30 公告明确：

- “吉事办”微信小程序首批支持长春市和吉林市的地铁公交；
- 其他“吉事办”移动端公告称可全省通行。

据此已补齐长春、吉林市的“吉事办”及微信小程序支持；对省内其他城市未在缺少城市级运营信息时直接推断微信支持。

PR：https://github.com/Ivysauro/CNRT/pull/153
依据：https://zsj.jl.gov.cn/zsdt/dt/202504/t20250430_3452672.html
